### PR TITLE
Update windows-whatsnew.md - hold for May 14

### PR DESCRIPTION
**Pull request by v-vtrieu**
@denisebmsft please review and approve for May's content.  Thank you.

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/windows-whatsnew.md [hunk 1]
```diff
@@ -38,6 +38,15 @@ All updates contain:
...
 
+## May-2023 (Release version: 10.8295.22621.1023)
+
+|OS  |KB  |Release version  |
+|---------|---------|---------|
+|Windows Server 2012 R2, 2016 |[KB 5005292](https://support.microsoft.com/en-us/topic/microsoft-defender-for-endpoint-update-for-edr-sensor-f8f69773-f17f-420f-91f4-a8e5167284ac)|10.8295.22621.1023|
+  
+**What's new**
+
+- Support new security settings management capabilities
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/windows-whatsnew.md]:
```suggestion
- Supports new security settings management capabilities
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-vtrieu, minor suggested change, please.

**Comment by denisebmsft** [comment]
> Copy review - @v-vtrieu, minor suggested change, please.

Done. Thank you @JohanFreelancer9 @v-vtrieu

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @v-vtrieu

**Pull request merged**